lib/results.go

```diff
@@ -217,6 +254,15 @@ func NewCSVDecoder(r io.Reader) Decoder {
 
 		r.Method = rec[9]
 		r.URL = rec[10]
+		if len(rec) > 11 {
```

Comment:
I think if we set `FieldsPerRecord = 12`, then this is always going to be true, or else an error would be returned by read.

> FieldsPerRecord is the number of expected fields per record. If FieldsPerRecord is positive, Read requires each record to have the given number of fields.

Reply:
Addressing this in a follow up commit. Thanks for the contribution!